Leaf API.postman_environment.json

```
@@ -186,9 +186,39 @@
 			"key": "trimble_application_name",
 			"value": "fill with your app's",
 			"enabled": true
+		},
+		{
+			"key": "raven_client_id",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_client_secret",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_redirect_url",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_code",
+			"value": "code after redirecting user",
+			"enabled": true
+		},
+		{
+			"key": "raven_refresh_token",
+			"value": "(auto)",
+			"enabled": true
```

I don't see the command line in step 2 to set the raven_refresh_token.
This property is defined using JS to take the response received and set this property.
We can also add a better value for this.
```suggestion
			"key": "raven_refresh_token",
			"value": "Raven refresh token obtained from step 2 "Get Raven Token" (auto)",
			"enabled": true
```

---

Done.